### ISSUE TITLE
Non-programmers

### ISSUE THREAD

**Issue by arisbanach** [x89/Shreddit#56]
Any chance you could create an executable file so that people can use this without needing terminal experience? If not, can you create an ELI5 walkthrough of how to use it for those who aren't familiar with the terminal?

**Comment by x89** [comment]
Unfortunately not, but I am considering developing a web app based on the same principal — now that there's OAuth2 you wouldn't need to provide a 3rd party website with your credentials, permissions are fine-grained and you can revoke app permissions at any time.

It's somewhere on the to-do list.
